docs/fence.md

```diff
@@ -0,0 +1,23 @@
+# Return traces along an arbitrary path
+
+Return slices along an arbitrary path of x,y coordinates. Such as along a
+wellbore. Coordinates can be specified in different coordinate system, and
+multiple interpolation methods are available. 
+
+## Response
+On success (200) the response consists of two parts, metadata and data.
+
+### Metadata part
+*Content-Type: application/json*
+Not implemented yet. Content-Length is 0.
+
+### Data part
+*Content-Type: application/octet-stream*
+A raw byte array containing the fence itself. The byte array needs to be parsed
```

Comment:
Note that this md file is user-facing and you use here hard breaks on 80.
From user point of view it looks a bit weird. I, as a user, would expect text to flow the whole width of the window.

Reply:
I have noticed, and I like it :D